pkg/sanity/cloudbackup.go

```diff
@@ -393,8 +392,9 @@ var _ = Describe("Cloud backup [OpenStorageCluster]", func() {
 					CredentialId: credID,
 				}
 
-				_, err := bc.Enumerate(context.Background(), enumerateReq)
-				Expect(err).To(HaveOccurred())
+				resp, err := bc.Enumerate(context.Background(), enumerateReq)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.GetBackups()).To(BeEmpty())
```

Comment:
based on description of the issue. we only address the "empty"  here right? what if the volume does have cloudbackups?